### PR TITLE
Update confirmation page to use card design from list page

### PR DESCRIPTION
**Pull request by jbalboni**
## Description
The info we show on the confirmation page should use the card design used on the appointment list page.

## Testing done
Local and unit testing

## Screenshots
Screen Shot 2020-01-06 at 5 19 34 PM
Screen Shot 2020-01-07 at 1 41 51 PM
Screen Shot 2020-01-07 at 1 42 50 PM
Screen Shot 2020-01-07 at 1 48 14 PM

## Acceptance criteria
- [ ] Card designs match list items as best we can

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/api/index.js [hunk 1]
```diff
@@ -276,7 +276,7 @@ export function getRequestLimits(facilityId, typeOfCareId) {
 export function getClinics(facilityId, typeOfCareId, systemId) {
   let promise;
   if (USE_MOCK_DATA) {
-    if (facilityId.includes('983')) {
+    if (facilityId === '983') {
```
Comment (jbalboni) [inline src/applications/vaos/api/index.js]: This not being exact was messing with the logic for matching clinic ids to past appointments

File: src/applications/vaos/containers/ConfirmationPage.jsx [hunk 2]
```diff
@@ -4,34 +4,49 @@ import PropTypes from 'prop-types';
...
+  fetchFacilityDetails,
+} from '../actions/newAppointment';
+import { FLOW_TYPES, FACILITY_TYPES } from '../utils/constants';
 import ConfirmationDirectScheduleInfo from '../components/ConfirmationDirectScheduleInfo';
 import ConfirmationRequestInfo from '../components/ConfirmationRequestInfo';
 
 export class ConfirmationPage extends React.Component {
+  componentDidMount() {
+    if (
+      !this.props.facilityDetails &&
```
Comment (jbalboni) [inline src/applications/vaos/containers/ConfirmationPage.jsx]: We might have the details already (from the clinic choice page) and we don't need them for CC, so don't fetch if either of those things are true.

File: src/applications/vaos/newAppointmentFlow.js [hunk 3]
```diff
@@ -180,8 +181,25 @@ export default {
           Sentry.captureException(error);
         }
 
-        dispatch(startDirectScheduleFlow(appointments));
-        return 'clinicChoice';
+        if (appointments) {
```
Comment (jbalboni) [inline src/applications/vaos/newAppointmentFlow.js]: This makes sure we have clinics to choose from after filtering them by past visited clinics

**Review by narin — APPROVED** [review]
LGTM.  Nice touch using react hooks 👍🏼

**Review by cvalarida — APPROVED** [review]

**Pull request merged**